tests/test_helper.rb

```diff
@@ -48,8 +48,8 @@ def assert_these_appear_before_these haystack, before, after
 module TLDRunner
   Result = Struct.new(:stdout, :stderr, :exit_code, :success?, keyword_init: true)
 
-  def self.should_succeed files, options = nil
-    run(files, options).tap do |result|
+  def self.should_succeed files, options = nil, ensure_time_bomb: false
```

Comment:
I've defaulted this to false so it's effectively "no change" across the entire test suite unless enabled (see suite_summary_test.rb), but I feel like it should be true by default, because we're testing TLDR, so we should expect to see the same as it's primary usage and output, which is for Devs, i.e. 1.8s time bomb.

What do y'all think? =)

Reply:
We ensure the time bomb in a couple other contrived ways based on sleeping -- I had a lot loaded in context then that I don't right now so I think I"m :-1: on changing it on a whim

Reply:
Ok cool, thanks =) Is it ok as is then to keep as-is, defaulting to false? Or is there a better way to allow the test added in tests/fixture/suite_summary_too_slow.rb to pass when run in GitHub Actions?

Reply:
It's probably fine the way it is, but to be honest we're surely triggering time expiring in CI already elsewhere, aren't we? How are we ensuring that?

Reply:
Not that I could find 🤔  so I wrote a new test file for it, as I couldn't find an existing one to append to.

I found tests/fixture/slow.rb but that was sleeping 400ms in each test, so when run parallel, it wouldn't have triggered time_bomb. That was what I thought at the time…

However I just took another look at it and can't see tests/fixture/slow.rb being called anywhere, or by the CI? But with the instructions at the top of the file, it _does_ trigger the time_bomb because there are more than 5 sleeps of 400ms, and giving a seed turns off parallelism by default (unless also requested with `--parallel`).

🤔

Reply:
I'm happy to change this however it needs to be changed – my addition of the suite_summary_too_slow fixture and test [caused CI to fail because it didn't use the time_bomb](https://github.com/tendersearls/tldr/actions/runs/7358198106/job/20031073067), so if there's an existing setup to ensure that test (or similar if it needs moving to a different file or format) can pass in CI, then I'm all for it =)